boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx

```diff
@@ -17,7 +17,7 @@ interface State {
  * class component because according to the documentation, only class
  * components can be error boundaries.
  *
- * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/Error-Boundary.md)
+ * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/concept/Error-Boundary.md)
```

Comment:
@mateoguzmana Thanks for the find! I think let's link directly to the hosted docs page now that it's up

https://docs.infinite.red/ignite-cli/concept/Error-Boundary/

Reply:
Nice, applied!